lib/metadata/include/irods/metadata.hpp

```diff
@@ -0,0 +1,86 @@
+#ifndef IRODS_METADATA_HPP
+#define IRODS_METADATA_HPP
+
+#include <string>
+#include <vector>
+#include <map>
+
+#include "irods/entity.hpp"
+#include "irods/rcConnect.h"
+
+#ifdef IRODS_METADATA_ENABLE_SERVER_SIDE_API
+#    include "irods/rsModAVUMetadata.hpp"
+using rxComm_t = rsComm_t;
+#    define rxModAVUMetadata rsModAVUMetadata
+#    define IRODS_FILESYSTEM_ENABLE_SERVER_SIDE_API
+#    define IRODS_QUERY_ENABLE_SERVER_SIDE_API
+#else
+#    include "irods/modAVUMetadata.h"
+using rxComm_t = rcComm_t;
+#    define rxModAVUMetadata rcModAVUMetadata
+#endif
+
+#include "irods/irods_query.hpp"
```

Comment:
Could/Should this be included in the .cpp file?

Reply:
The header doesn't look like it has any types that require `irods_query.hpp` but I'm currently building to prove myself right (or wrong)